remoulade/api/app/__init__.py

```diff
@@ -0,0 +1,27 @@
+""" Creating the instance of the app """
+from flask import Flask
+
+from remoulade.state import State
+from typing import Dict, List
+from .config import Config
+
+
+class MessageAdmin:
```

Comment:
I would prefer you do not use this object, I make things more complex than it should be. We should read the code directly in the route, making things more clear.